jsonschema/golangci.next.jsonschema.json

```diff
@@ -422,33 +422,6 @@
           "default": [],
           "examples": [["mytag"]]
         },
-        "skip-dirs": {
```

Comment:
There's a section in the JSON schema regarding deprecations: https://json-schema.org/draft/2019-09/json-schema-validation#rfc.section.9.3

Should we use that instead of just removing the configuration?

Reply:
The deprecation doesn't work very well: there is no warning during validation, so people will still use them.

Reply:
FYI we are using Draft-07 (as recommended by the schema store)

https://json-schema.org/draft-07/json-schema-validation

There is no `deprecated` properties.

Reply:
FYI I tried `deprecated` and `deprecationMessage`, and there are no effects.